### ISSUE TITLE
Problem with triplet loss

### ISSUE THREAD

**Issue by shayan-aqabarary** [KaiyangZhou/deep-person-reid#531]
I have 2 changes in original code, first change network from "resnet" to "osnet" (osnet_x1_0). second change loss function from "softmax" to "triplet". and I get this error. can anyone help me?
@KaiyangZhou 
![Screenshot (8)](https://user-images.githubusercontent.com/37603477/205451442-26220c01-b00f-4a94-be60-5c027417262b.png)

**Comment by yajiedesign** [comment]
use ImageTripletEngine
